condor/samples/hhh6bPNetAK4_2018_MC.yaml

```diff
@@ -123,3 +123,113 @@ dy:
 #    xs: DYJetsToLL_Zpt-650ToInf_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8
 
 
+whh:
```

Comment:
@xygeng1 could you triple check that now all samples that we need are on the lists of all the years?